OpenRA.Mods.Common/Traits/SupportPowers/AirstrikePower.cs

```diff
@@ -63,8 +57,8 @@ public AirstrikePower(Actor self, AirstrikePowerInfo info)
 
 		public override void SelectTarget(Actor self, string order, SupportPowerManager manager)
 		{
-			if (info.UseDirectionalTarget)
-				self.World.OrderGenerator = new SelectDirectionalTarget(self.World, order, manager, Info.Cursor, info.DirectionArrowAnimation, info.DirectionArrowPalette);
+			if (info.UseDirectionalTarget && Info is DirectionalSupportPowerInfo directionalSupportPowerInfo)
```

Comment:
Is this needed? `AirstrikePowerInfo` explicitly inherits `DirectionalSupportPowerInfo` so it should be fine to pass it directly.

Reply:
Or perhaps better to move this method into `DirectionalSupportPower`?